packages/vscode-extension/src/commands/ListFiles.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ ** Copyright (c) 2021 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+import { window } from 'vscode';
+import { COMMAND, LIST_FILES } from '../service/TranslationKeys';
+import { actionResultStatus, FolderUtils } from '../util/ExtensionUtil';
+
+import BaseAction from './BaseAction';
+import { FolderType } from './FolderType';
+
+const COMMAND_NAME = 'listfiles';
+
+const LIST_FILES_COMMAND = {
+	OPTIONS: {
+		FOLDER: 'folder',
+	},
+};
+
+export default class ListFiles extends BaseAction {
+	constructor() {
+		super(COMMAND_NAME);
+	}
+
+	protected async execute(): Promise<void> {
+		const fileCabinetFolders = await this._getListFolders();
+		const filteredFolders = this.filterDisabledFolders(fileCabinetFolders);
+		if (filteredFolders === undefined) {
```

Comment:
Will there be a case where filteredFolders can be null? If there is, it'll be better if the checking here is if (filteredFolders == null).

Reply:
I think it would be only undefined. However I could add the null case... eslint makes me use ===,

Reply:
I think if (filteredFolders == null) should cover both null and undefined.

Reply:
Yes, I know... but eslint complaints about it. If you want we could discuss it in the daily

Reply:
I see. Ok, if you're sure that its' just undefined, then I'm ok with it. But if you think null might happen, please add the null case and use === (so no eslint warning).